tutorials/plot_ica_from_raw.py

```diff
@@ -40,7 +40,9 @@
 ###############################################################################
 # 1) Fit ICA model using the FastICA algorithm.
 
-# Other available choices are `infomax` or `extended-infomax`
+# Other available choices are `picard`, `infomax` or `extended-infomax`.
+# Picard is generally faster and gives more consistent results than the other
+# algorithms.
```

Comment:
Probably here and in the `ICA` docstring you should mention the same tradeoffs you've talked about in these PR comments, i.e. something about to get roughly equivalent convergence behavior one tolerance should be the square of another, and that if these are set equal `picard` is faster and (maybe) still more stable, etc.

Reply:
I agree, this is important. Right now it is far too general and might confuse more than explain.